alica_engine/include/engine/allocationauthority/EntryPointRobotPair.h

```diff
@@ -38,6 +36,6 @@ struct hash<alica::EntryPointRobotPair>
     typedef alica::EntryPointRobotPair argument_type;
     typedef std::size_t value_type;
 
-    value_type operator()(argument_type& eprp) const { return std::hash<long int>()(eprp.getEntryPoint()->getId()) + eprp.getRobot()->hash(); }
+    value_type operator()(argument_type& eprp) const { return std::hash<long int>()(eprp.getEntryPoint()->getId()) + eprp.getRobot(); }
```

Comment:
need to hash `eprp.getRobot()` as well i.e. `std::hash<AgentID>(eprp.getRobot())`

Reply:
addition is also a poor way to combine hashes. Can use code similar to boost::hash_combine: http://myeyesareblind.com/2017/02/06/Combine-hash-values/